src/tools/miri/src/concurrency/thread.rs

```diff
@@ -272,8 +274,9 @@ impl Time {
     fn get_wait_time(&self, clock: &Clock) -> Duration {
         match self {
             Time::Monotonic(instant) => instant.duration_since(clock.now()),
-            Time::RealTime(time) =>
-                time.duration_since(SystemTime::now()).unwrap_or(Duration::new(0, 0)),
+            Time::RealTime(time) => {
+                time.duration_since(SystemTime::now()).unwrap_or(Duration::new(0, 0))
+            }
```

Comment:
This and the diff below look like a repeat of https://github.com/rust-lang/rust/pull/110539#discussion_r1177614183

Reply:
Fixed formatting.